src/cloudinary-image.component.ts

```diff
@@ -145,4 +152,16 @@ export class CloudinaryImage
     })
     this.placeholderComponent.options = placeholderOptions;
   }
+
+  accessibilityModeHandler() {
+    const accessibilityEffect = {
+      'darkmode': {effect: 'tint:75:black'},
+      'brightmode': {effect: 'tint:50:white'},
+      'monochrome': {effect: 'grayscale'},
+      'colorblind': {effect: 'assist_colorblind'}
+    }
```

Comment:
Please take this out of here

Reply:
separate file?

Reply:
@strausr, either to a const area in this file or to consts file. 
In the end this should move to the js-core so other FW will be able to use the same pre-defined transformations